fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/bridge/facades/SimulationFacade.java

```diff
@@ -619,6 +681,10 @@ private InductionLoopInfo processInductionLoopSubscriptionResult(long time, Indu
                 .create();
     }
 
+    private AgentData processPersonSubscriptionResult(long time, PersonSubscriptionResult result) {
+        return new AgentData(time, result.id, result.position.getGeographicPosition(), null, null, AgentData.TripStatus.WALKING);
```

Comment:
I'm unsure if we should extend AgentData with speed and heading, since we would not use that for the message sending use-case.